app/controllers/search_controller.rb

```diff
@@ -4,23 +4,7 @@ class SearchController < ApplicationController
   # You can implement pattern searching using LIKE clause
   # Example : SELECT * FROM patients WHERE first_name LIKE "%pattern%";
 
-  def with_read_only_connection
```

Comment:
Makes sense! Me @abhinandanarya06 and @peeyush14goyal discussed that this is probably not a good idea.